hxd/System.hl.hx

```diff
@@ -158,6 +158,9 @@ class System {
 		#if usesys
 		haxe.System.reportError(err + stack);
 		#else
+		if ( Sys.systemName() != 'Windows' )
+			return;

```

Comment:
Return should be after printing of the error to the system console, not before.

Reply:
That was my initial thought but then the error information is printed twice:

```
hl freeze_bug.hl
Null access
Called from $Main.main (Main.hx line 14)
Uncaught exception: Null access
Called from $Main.main(Main.hx:14)
Called from fun$3997(?:1
```

Reply:
And BTW the error should be printed on stderr, not stdout, but that's orthogonal to the subject of this PR.